### ISSUE TITLE
Add main entry point for css/sass files

### ISSUE THREAD

**Issue by mebibou** [apnsngr/bulma-popover#3]
It would be nice to be able to do
```sass
@import '~bulma-popover';
```
but this doesnt work as there are a couple of things missing from the `package.json`:
```json
{
  "main": "css/bulma-popver.min.css",
  "style": "css/bulma-popver.min.css",
  "sass": "bulma-popover.scss",
}
```

Note: it would be nice to fix the typo and have `bulma-popover.min.css` instead of `bulma-popver.min.css`

**Comment by apnsngr** [comment]
Hello! I really appreciate your feedback. I am going to make this update and the others that you suggested this evening.

**Comment by apnsngr** [comment]
My node-sass install is not working on my laptop I believe it is due to some macOS Homebrew stuff. I will make this update tomorrow on a different computer.

**Comment by apnsngr** [comment]
I have made the update and published it as version 1.0.2 (maybe it should have been v2 since it is technically a breaking change). I was able to get my personal laptops dev environment sorted out so I will be try to make the other changes you requested this weekend.

**Comment by mebibou** [comment]
@apnsngr yes should have been `2.0.0`
